draft-pauly-privacypass-auth-scheme.md

```diff
@@ -203,23 +203,23 @@ following attributes:
 value. This MUST be unique for every 401 HTTP response to prevent replay attacks.
 This attribute is required for all challenges.
 
-- "issuer-key", which contains a base64url encoding of the SubjectPublicKeyInfo object
-for use with the issuance protocol indicated by the challenge. This attribute MAY
+- "token-issuance-key", which contains a base64url encoding of the public key for
```

Comment:
How about:

```suggestion
- "token-validation-key", which contains a base64url encoding of the public key for
```

Or

```suggestion
- "token-key", which contains a base64url encoding of the public key for
```

I may lean towards the short "token-key" since this is sent on the wire, and in the challenge, it's only the verification key that matters.

Reply:
Yeah, based on the other issue chat, I think token-validation-key is best. I'll update this soon to take that into account.